setup.sh

```diff
@@ -172,7 +172,14 @@ function _adduser() {
   fi
   chmod 750 /home/${user}
   if grep ${user} /etc/sudoers.d/swizzin >/dev/null 2>&1 ; then echo "No sudoers modification made ... " ; else	echo "${user}	ALL=(ALL:ALL) ALL" >> /etc/sudoers.d/swizzin ; fi
-  echo "D /var/run/${user} 0750 ${user} ${user} -" >> /etc/tmpfiles.d/${user}.conf
+  
+  if [[ $codename =~ "focal" ]] ; then
+    echo "D /run/${user} 0750 ${user} ${user} -" >> /etc/tmpfiles.d/${user}.conf
+  else
+    echo "D /var/run/${user} 0750 ${user} ${user} -" >> /etc/tmpfiles.d/${user}.conf
+
+  fi
+  
```

Comment:
I believe this can just be applied to all supported distros and it's not needed to only do for focal. That said, this change needs to be reflected in the rtorrent script for the unix socket.

Reply:
It seems that this just runs fine with this change on my system, without a modification on the rtorrent side

Reply:
When /var/run is deprecated this will cause a failure, *eventually*